src/components/Header/Header.css

```diff
@@ -0,0 +1,13 @@
+.Header{
+	overflow: auto;
+}
+.Header.dark{
+	background: #333;
+	color: #fff;
+}
+.Header .SettingsWheel{
+	float: right;
+}
+.Header__content{
```

Comment:
All css selectors in `Header.css` should begin with `.Header`

Reply:
`.Header__content` should be `.Header .Header__content`

Reply:
Given what I know about BEM - http://getbem.com/introduction/  I'm inclined to disagree. But this isn't my field of expertise

@peasandwell @ParkyGames - Could you help fill the gaps with BEM and the context where it's being used here.

Reply:
I agree with what @ShaunBulbrook said, if we're following BEM for the entire project then this line is fine.

Reply:
That's something we should probably settle. :) 
Any thoughts on using/not-using?

Reply:
I don't see the point of BEM in React, BEM seems to be solving the issue of global styles by putting everything within a "Block" and grouping them accordingly - We're already solving this by putting a top-level component classname. If y'all want to use BEM internally on a component that's fine, but I still think all styles should be prefixed with the classname for the component to maintain component scoping

Reply:
Plus if your React component is big enough to need BEM to keep the styles organised it should probably be broken into subcomponents

Reply:
It's hard to say right now because BEM is suited for larger projects, mainly if the CSS is in one large file with components flying everywhere. While this project is due to expand, I don't think it will get complicated enough to warrant the need for something like BEM.

My vote is "nay"